be-traveltrove/src/main/java/com/traveltrove/betraveltrove/utils/DatabaseLoader.java

```diff
@@ -146,25 +147,68 @@ public void loadEvents() {
     public void loadTours() {
         List<Tour> sampleTours = List.of(
                 Tour.builder()
-                        .id(null)
+                        .id(UUID.randomUUID().toString())
                         .tourId("TOUR01")
-                        .name("Spring Festival")
-                        .description("A vibrant festival celebrating the arrival of spring.")
+                        .name("Arabian Desert Safari")
+                        .description("Embark on a luxurious desert adventure with private 4x4 rides, camel treks, and an evening of stargazing in Bedouin-style camps.")
                         .build(),
                 Tour.builder()
-                        .id(null)
+                        .id(UUID.randomUUID().toString())
                         .tourId("TOUR02")
-                        .name("Tech Expo 2024")
-                        .description("Showcasing the latest advancements in technology.")
+                        .name("Splendors of Dubai")
+                        .description("Experience the glamour of Dubai with exclusive tours of Burj Khalifa, Palm Jumeirah, luxury shopping, and a private yacht cruise.")
                         .build(),
                 Tour.builder()
-                        .id(null)
+                        .id(UUID.randomUUID().toString())
                         .tourId("TOUR03")
-                        .name("Wine Tasting Retreat")
-                        .description("A relaxing getaway featuring fine wines and scenic views.")
+                        .name("Qatar Cultural Retreat")
+                        .description("Discover Qatar’s rich heritage with visits to Souq Waqif, the Museum of Islamic Art, and luxury desert escapes.")
+                        .build(),
+                Tour.builder()
+                        .id(UUID.randomUUID().toString())
+                        .tourId("TOUR04")
+                        .name("Oman’s Hidden Gems")
+                        .description("Explore the beauty of Oman with a mix of luxury and tradition, including Wadi Shab, Musandam fjords, and opulent resorts in Muscat.")
+                        .build(),
```

Comment:
I see that you've changed the `id` to an UUID, but I was talking about the `tourId`, the `id` is automatically generated by MongoDB when it's null so it should remain null. I've talked with Adem and we should stick to using UUIDs for the public ids like `tourId`.